src/custom-elements/src/components/expansion-panel/expansion-panel.tsx

```diff
@@ -1,6 +1,7 @@
 import { Component, Prop, Watch, h, Element, State } from '@stencil/core';
```

Comment:
State is no longer used

```suggestion
import { Component, Prop, Watch, h, Element } from '@stencil/core';
```